Azure/Azure-Sentinel
# Windows event batch3 v2

-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
Added WindowsEvent data to the detections:
powershell_empire
GainCodeExecutionADFSviaWMI
ADFS-DKM-MasterKey-Export
AADHostLoginCorrelation
EmailAccessviaActiveSync
ExchangeServerVulnerabilitiesMarch2021IoCs
SUNSPOTLogFile

   Reason for Change(s):
Added WindowsEvent data in addition to securityEvent

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
manual testing
_Note: If updating a detection, you must update the version field._

> After the submission has been made, please look at the Validation Checks.

   Checked that the validations are passing and have addressed any issues that are present:
   -Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------
